certified-connectors/Scancloud/apiDefinition.swagger.json

```diff
@@ -154,8 +154,8 @@
             "type": "string",
             "default": "",
             "required": false,
-            "description": "Secplugs API Key",
-            "x-ms-summary": "Secplugs API Key"
+            "description": "Scancloud API Key",
+            "x-ms-summary": "Scancloud API Key"
```

Comment:
@Ajaatsecplugs please don't copy paste summaries and descriptions, they should differ. Can you please make a bit more detailed descriptions? You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).